wallpaper-bindings/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+use more_wallpapers::Mode;
```

Comment:
Also can we include this at the very top? It sets up the default clippy lint level for the crate. You may need to resolve some lint errors after adding this:

```rust
#![deny(clippy::all, clippy::pedantic, rust_2018_idioms, clippy::unwrap_used)]

```

Reply:
Actually since [rust 1.74.0](https://blog.rust-lang.org/2023/11/16/Rust-1.74.0.html) you can define lints in the Cargo.toml. I'll add it to the workspace one in #17, can you just add 

```toml
[lints]
workspace = true
```

to your crate's Cargo.toml?